flume-ng-sinks/flume-ng-kudu-sink/pom.xml

```diff
@@ -159,6 +159,7 @@
       <activation>
         <os>
           <family>Unix</family>
+          <arch>amd64</arch>
```

Comment:
At the moment there is kudu-binary.jar only for Linux and Mac x86_64: https://repo1.maven.org/maven2/org/apache/kudu/kudu-binary/1.14.0/